src/instructlab/model/backends/backends.py

```diff
@@ -39,6 +39,7 @@
 API_ROOT_WELCOME_MESSAGE = "Hello from InstructLab! Visit us at https://instructlab.ai"
 CHAT_TEMPLATE_AUTO = "auto"
 CHAT_TEMPLATE_TOKENIZER = "tokenizer"
+MAX_STABLE_VRAM_WAIT = 30
```

Comment:
Are we sure 30 is enough here? Would it be worth the extra work to try to make this configurable? Both eval and phased training are using `launch_server()` in evaluate.py so we'd just need to pass the value from config into that.

Reply:
good question, i was on the fence on that one. There might be a scenario we would want to change it, but i haven't seen it yet. The timing is a fairly small window since it's a few seconds after process exit colliding with the very early start phase of vllm.  My thinking is we could add that as a follow-up if needed, and get the simple constant version in for now to get in our images and e2e testing.

Reply:
Are you seeing any cases that don't succeed in the 5 stable check loop below?

Reply:
In my other comments on this PR, I was wondering whether we could remove the magic number 5 with a check of free memory is not changing for a couple iterations.  I don't see a good reason to make the 30 secs configurable unless we see something useful happening >10-15 secs or we get a lot cases that don't exit out with the 5 stable check loop.